README.md

```diff
@@ -225,6 +225,25 @@ module.exports = function(environment) {
 ```hbs
 {{imgix-image path="/test.png" disableLibraryParam={true} }}
 ```
+### imgix-bg
+
+This component will render a `div` whose `background-image` is set to the passed in image path. Content can be added within the `imgix-bg` tags and the component will automatically resize to fit around it.
+
+```hbs
+{{#imgix-bg path='/users/1.png' }}
+  Some content here
+{{/imgix-bg}}
+```
+
+Which will generate html similar to the following:
+
+```html
+<div id="ember226" alt="" style="background-image: url('https://my-social-network.com/users/1.png?fit=crop&w=1246&h=15&dpr=2&ixlib=ember-2.0.0');background-size: cover" class="imgix-bg ember-view">    
```

Comment:
Let's distill this example a little bit by removing the `id` and `alt` attributes.